util/topgen/top_uvm_reg.sv.tpl

```diff
@@ -76,14 +69,14 @@ package chip_ral_pkg;
 
   windows = top.window_block.windows
 %>\
-${make_ral_pkg_hdr(dv_base_prefix, if_packages)}
+${make_ral_pkg_hdr(dv_base_names[4], if_packages)}
```

Comment:
if some base classes are dv_base_*, some are ast_base_*, we need to import both pkg.

Reply:
I believe this is handled at util/reggen/uvm_reg_base.sv.tpl  line 197?

Reply:
looks good, thanks